bootcamp_calculator.py

```diff
@@ -23,7 +23,10 @@ def multiply(x, y):
 
     @staticmethod
     def divide(x, y):
-        return x / y
+        if y == 0:
+            return "Validation error. You can’t divide by 0…"
+        else:
+            return round((x / y), 3)
 
```

Comment:
You have rounded only divide() method. What about others? add(), subtract(), multiply() also should be rounded to 3 decimals

Reply:
Done